doc/edam/api.rst

```diff
@@ -254,6 +255,67 @@ Field Name       Type                  Description
 part             String                Device identifier. e.g. *xc7a35tcsg324-1*
 ================ ===================== ===========
 
+vunit
+~~~~~~
+
+================ ===================== ===========
+Field Name       Type                  Description
+================ ===================== ===========
+vunit_options    List of String        Extra options for the VUnit test runner
+add_libraries    List of String        A list of framework libraries to add. Allowed values include "array_util", "com", "json4hdl", "osvvm", "random", "verification_components"
+vunit_runner     String                Name of the Python file exporting a ``VUnitRunner`` class (must derive from ``edalize.vunit_hooks.VUnitHooks``) that is used to configure and execute test. This allows very customized test control via VUnit's Python-interfaces.
+================ ===================== ===========
+
+In case a more advanced VUnit configuration or execution of the testbench is necessary, the option ``vunit_runner`` can be used to specify the filename of a Python-script which can hook into the construction, parametrization, and execution of the test runner. For this to work, the Python-skript must export a ``class VUnitRunner(vunit_hooks.VUnitHooks)`` which derives from (and optionally overrides) the behavior of ``vunit_hooks.VUnitHooks``.
+
+.. code-block:: python
+
+    from vunit_hooks import VUnitHooks as VH
+    from vunit.ui import Library, Results
+    from typing import Mapping, Collection
+
+    class VUnitRunner(VUnitHooks):
+        '''
+        Example of custom VUnit instrumentation
+        '''
```

Comment:
Nit: https://www.python.org/dev/peps/pep-0257/#id16 suggests formatting such docstrings as
``"""Example of custom VUnit instrumentation."""``
(If you haven't tried https://pypi.org/project/black/ - it's awesome!)

Reply:
Thanks for pointing out the black tool! Did not come across this yet. Although it did not help me in formatting the docstring. Still had to do it manually... or maybe I'm just using it wrong.